Eigen3 not found on Linux with Travis-CI

From Travis-CI
```
CMake Error at CMakeLists.txt:94 (find_package):
  Could not find a configuration file for package "Eigen3" that is compatible
  with requested version "3.0".
  The following configuration files were considered but not accepted:
    /usr/lib/cmake/eigen3/Eigen3Config.cmake, version: unknown
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -47,24 +47,24 @@ matrix:
         - CXX=g++-7
         - LIBLA=arma
 
-#    - os: linux
-#      dist: xenial
-#      addons:
-#        apt:
-#          sources:
-#            - ubuntu-toolchain-r-test
-#          packages:
-#            - liblapack-dev
-#            - libblas-dev
-#            - libboost-dev
-#            - libeigen3-dev
-#            - g++-7
-#            - lcov
-#          update: true
-#      env:
-#        - CC=gcc-7
-#        - CXX=g++-7
-#        - LIBLA=eigen
+    - os: linux
+      dist: xenial
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - liblapack-dev
+            - libblas-dev
+            - libboost-dev
+            - libeigen3-dev
+            - g++-7
+            - lcov
+          update: true
+      env:
+        - CC=gcc-7
+        - CXX=g++-7
+        - LIBLA=eigen
 
 script:
   - bash .travis/build.sh

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,7 +3,7 @@ language: cpp
 matrix:
   include:
     - os: osx
-      osx_image: xcode10.1
+      osx_image: xcode11.2
       env:
         - LIBLA=arma
       addons:
@@ -16,7 +16,7 @@ matrix:
           update: true
 
     - os: osx
-      osx_image: xcode10.1
+      osx_image: xcode11.2
       env:
         - LIBLA=eigen
       addons:
@@ -29,7 +29,7 @@ matrix:
           update: true
 
     - os: linux
-      dist: xenial
+      dist: bionic
       addons:
         apt:
           sources:
@@ -48,7 +48,7 @@ matrix:
         - LIBLA=arma
 
     - os: linux
-      dist: xenial
+      dist: bionic
       addons:
         apt:
           sources:

--- a/.travis.yml
+++ b/.travis.yml
@@ -35,6 +35,7 @@ matrix:
           sources:
             - ubuntu-toolchain-r-test
           packages:
+            - cmake
             - liblapack-dev
             - libblas-dev
             - libboost-dev
@@ -54,6 +55,7 @@ matrix:
           sources:
             - ubuntu-toolchain-r-test
           packages:
+            - cmake
             - liblapack-dev
             - libblas-dev
             - libboost-dev

--- a/include/libirc/linalg.h
+++ b/include/libirc/linalg.h
@@ -4,9 +4,9 @@
 #ifdef HAVE_ARMA
 #include <armadillo>
 #elif HAVE_EIGEN3
-#include <Eigen/Dense>
-#include <Eigen/Geometry>
-#include <Eigen/LU>
+#include <eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Geometry>
+#include <eigen3/Eigen/LU>
 #else
 #error
 #endif

--- a/src/test/atom_test.cpp
+++ b/src/test/atom_test.cpp
@@ -11,7 +11,7 @@
 #include <armadillo>
 using arma::vec3;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 #else
 #error

--- a/src/test/connectivity_test.cpp
+++ b/src/test/connectivity_test.cpp
@@ -20,7 +20,7 @@ using mat = arma::mat;
 template<typename T>
 using Mat = arma::Mat<T>;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/io_test.cpp
+++ b/src/test/io_test.cpp
@@ -10,7 +10,7 @@ using arma::mat;
 using arma::vec;
 using arma::vec3;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/irc_test.cpp
+++ b/src/test/irc_test.cpp
@@ -17,7 +17,7 @@ using mat = arma::mat;
 template<typename T>
 using Mat = arma::Mat<T>;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/linalg_test.cpp
+++ b/src/test/linalg_test.cpp
@@ -8,7 +8,7 @@ using vec3 = arma::vec3;
 using vec = arma::vec;
 using mat = arma::mat;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/mathtools_test.cpp
+++ b/src/test/mathtools_test.cpp
@@ -12,7 +12,7 @@ using mat = arma::mat;
 template<typename T>
 using Mat = arma::Mat<T>;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/molecule_test.cpp
+++ b/src/test/molecule_test.cpp
@@ -11,7 +11,7 @@
 #include <armadillo>
 using arma::vec3;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 #else
 #error

--- a/src/test/transformation_test.cpp
+++ b/src/test/transformation_test.cpp
@@ -20,7 +20,7 @@ using mat = arma::mat;
 template<typename T>
 using Mat = arma::Mat<T>;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

--- a/src/test/wilson_test.cpp
+++ b/src/test/wilson_test.cpp
@@ -18,7 +18,7 @@ using vec3 = arma::vec3;
 using vec = arma::vec;
 using mat = arma::mat;
 #elif HAVE_EIGEN3
-#include <Eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Dense>
 using vec3 = Eigen::Vector3d;
 using vec = Eigen::VectorXd;
 using mat = Eigen::MatrixXd;

